### ISSUE TITLE
Cosmos is throwing Missing Reference error and is unbale to recognize namespaces and basic C# libraries like System

### ISSUE THREAD

**Issue by shreyas13579** [CosmosOS/Cosmos#1167]
#### Area of Cosmos - What area of Cosmos are we dealing with?
I am trying to create an OS in Cosmos User Kit. I am using the latest version of Cosmos that came out in May 2019 (this year) and Visual Studio 2019. When I open a new Cosmos Kernel, I get auto generated code by Cosmos, but on running it I get an Error Box full of errors (22 actually) asking me if I missed any references. 

#### Expected Behaviour - What do you think that should happen?
The code should run and it should appear in a OS emulator window.

#### Actual Behaviour - What unexpectedly happens?
I get a Error Box full of errors about Missing references and Cosmos is unable to recognize the Cosmos namespace and basics things like ``System``, ``Console``, etc

#### Reproduction - How did you get this error to appear?
I just clicked the green button in VS2019 having my project name on it.

#### Version - Were you using the User Kit or Dev Kit? And what User Kit version or Dev Kit commit (Cosmos, IL2CPU, X#)?
I am using  Cosmos UserKit20190508 (the latest release).

This is what the errors look like:
![system_error](https://user-images.githubusercontent.com/43932047/57675827-a9302500-7640-11e9-98a0-66a85cf35018.png)

This is the code auto-generated by Cosmos:
![cosmos_code](https://user-images.githubusercontent.com/43932047/57675877-c9f87a80-7640-11e9-8bfe-8ee5bd7fc104.png)

**Comment by quajak** [comment]
Can you look under manage nuget packages and see if the "Cosmos Local Package Feed" package source
lists any package. If yes please run a nuget restore.

**Comment by shreyas13579** [comment]
@quajak After going into Manage NuGet Packages, I am getting this screen:
![nuget_cosmos](https://user-images.githubusercontent.com/43932047/57700916-828de080-7678-11e9-9d6b-967be8b0e53d.png)


What do I do?

**Comment by quajak** [comment]
Ypu have to select Include Prerelease

**Comment by shreyas13579** [comment]
@quajak 
I followed your instructions and updated the NuGet Packages. I also installed few other packages which appeared in the "Browse" tab and had Cosmos logo on them. Thankfully, all the errors disappeared. However, a new error appeared demanding a ``Main()`` method for "suitable entry point".

This is the screenshot for it:
![Main()_cosmos](https://user-images.githubusercontent.com/43932047/57721149-17a4cf80-76a1-11e9-9f7b-0ce346aac9fd.png)

**I have opened another issue for it at: https://github.com/CosmosOS/Cosmos/issues/1168**
